src/NServiceBus.Core.Tests/AssemblyScanner/When_scanning_assemblies_with_circular_dependencies.cs

```diff
@@ -14,10 +13,15 @@ public void ReferencesNServiceBus_circular()
         {
             // Assemblies already exist in TestDlls/circular:
             var circularDirectory = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestDlls\circular");
+
+            var classLibA = Path.Combine(circularDirectory, "ClassLibraryA.dll");
             var classLibB = Path.Combine(circularDirectory, "ClassLibraryB.dll");
 
-            // Put ClassLibraryB.dll in CurrentDomain.BaseDirectory so it resolves correctly
-            var tempClassLibB = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "ClassLibraryB.dll");
+            // Put assemblies in TestContext.CurrentContext.TestDirectory so they resolve correctly
```

Comment:
I wonder why're not pointing the assembly scanner directly to the circular subfolder instead?